engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -30,17 +30,17 @@
     private static final String DEFAULT_FRAME_BUFFER_NAME = "display";
     private static BindFBO defaultInstance = new BindFBO();
     private String fboName;
+    private FrameBuffersManager frameBuffersManager;
     private int fboId;
     private BindFBOTask task;
-    private FrameBuffersManager frameBuffersManager;
 
     public BindFBO(String fboName, FrameBuffersManager frameBuffersManager) {
         this.fboName = fboName;
         this.frameBuffersManager = frameBuffersManager;
         fboId = frameBuffersManager.getFBO(fboName).fboId;
     }
 
-    private BindFBO() {
+    public BindFBO() {
         this.fboName = DEFAULT_FRAME_BUFFER_NAME;
         fboId = DEFAULT_FRAME_BUFFER_ID;
     }
```

Comment:
I'd like to remove this method. Just write

`private static BindFBO defaultInstance = new BindFBO(DEFAULT_FRAME_BUFFER_NAME);`

and also

```
public BindFBO(String fboName) {
    this.fboName = fblName;
    if (fboName != DEFAULT_FRAME_BUFFER_NAME) {
        fboId = frameBuffersManager.getFBO(fboName).fboId;
        // also add here functionality to subscribe to the FBM!
    } else {
        fboId = DEFAULT_FRAME_BUFFER_ID;
    }
}
```